frontend/src/component/feature/FeatureView/FeatureVariants/FeatureEnvironmentVariants/EnvironmentVariantsModal/VariantForm/VariantForm.tsx

```diff
@@ -262,7 +262,7 @@ export const VariantForm = ({
 
     const validatePayload = (payload: IPayload) => {
         if (!isValidPayload(payload)) {
-            setError(ErrorField.PAYLOAD, 'Invalid JSON.');
+            setError(ErrorField.PAYLOAD, 'Invalid value.');
```

Comment:
Mayebe we can use the payload.type to specify "Invalid JSON" or "Invalid Number"